### ISSUE TITLE
RNApuzzler headers at wrong folder

### ISSUE THREAD

**Issue by martin-raden** [ViennaRNA/ViennaRNA#55]
Hi,

I installed Vienna v2.4.12 using 

```
 ./configure --prefix=[..]/ViennaRNA/2.4.12--without-doc --without-python --without-python3 --without-check --without-cluster --without-forester --without-json --without-svm --disable-unittests --without-kinwalker --without-tutorial  --without-tutorial-pdf  --without-rnalocmin --disable-shared --enable-static --without-gsl --without-swig --without-perl
```

when compiling pourRNA against the lib, I get

```
In file included from [..]/ViennaRNA/2.4.12//include/ViennaRNA/PS_dot.h:15:0,
                 from SC_DotPlot.h:15,
                 from SC_DotPlot.cpp:8:
[..]/ViennaRNA/2.4.12//include/ViennaRNA/plotting/layouts.h:24:10: fatal error: ViennaRNA/plotting/RNApuzzler/RNAturtle.h: No such file or directory
 #include "ViennaRNA/plotting/RNApuzzler/RNAturtle.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

when inspecting the installation folder the respective file localized in its root folder and the "RNApuzzler" folder is missing:

```
 --> ls -l [..]/ViennaRNA/2.4.12/include/ViennaRNA/plotting/
total 92
drwxr-sr-x   2 4096 May 20 13:43 ./
drwxr-sr-x  11 4096 May 20 13:36 ../
-rw-r--r--+  1 3675 May 20 13:36 alignments.h
-rw-r--r--+  1 21362 May 20 13:42 layouts.h
-rw-r--r--   1 20480 May 20 13:42 .layouts.h.swp
-rw-r--r--   1 3904 May 20 13:36 naview.h
-rw-r--r--   1 4706 May 20 13:36 probabilities.h
-rw-r--r--   1 5639 May 20 13:36 RNApuzzler.h
-rw-r--r--   1 3620 May 20 13:36 RNAturtle.h
-rw-r--r--+  1 6861 May 20 13:43 structures.h
-rw-r--r--+  1 1516 May 20 13:36 utils.h
```

so there seems to be an issue with the Makefile..

same for `stuctures.h` in the same folder. I fixed it for my stuff by editing the two headers, but I guess that is not the plan.. ;)

Thanks for investigating,
Martin

**Comment by RaumZeit** [comment]
Dear Martin,
thank you for reporting this!
We are aware of this issue and already have a fix for the next release. Indeed, the headers are installed to the wrong subdirectory due to an error in the corresponding Makefile.am.
We expect to release the next version 2.4.13 that, among other things, fixes this bug next week.

**Comment by RaumZeit** [comment]
Fixed since version 2.4.13
